integration_tests/utils.py

```diff
@@ -137,6 +137,30 @@ def wait_for_ipc(path, timeout=40.0):
             )
 
 
+def w3_wait_for_block(w3, height, timeout=240):
+    for i in range(timeout * 2):
```

Comment:
why multiply with 2?

Reply:
because it sleep `0.5` seconds, multiply the loop count with 2 make total time closer to timeout as seconds.